web/controllers/post_controller.ex

```diff
@@ -25,7 +25,7 @@ defmodule Healthlocker.PostController do
   end
 
   def index(conn, _params) do
-    posts = Repo.all(Post)
+    posts = Repo.all(from p in Post, where: ilike(p.content, "%#story%"))
```

Comment:
> It'd be nice to move this query into the model, something like this: https://blog.drewolson.org/composable-queries-ecto/

I read through the post and I understand how to do it, but can you explain what the benefit is of doing it that one? Is it easier to write tests, or can the tests be better in some way?